appinventor/buildserver/src/com/google/appinventor/buildserver/resources/runtime.scm

```diff
@@ -943,6 +943,41 @@
   ;;      )
   ;;  '(100 200 17 300))
 
+(define-syntax map_nondest
+  (syntax-rules ()
+    ((_ lambda-arg-name body-form list)
+     (yail-list-map (lambda (lambda-arg-name) body-form) list))))
+
+;;(define-syntax map_nondest
+;;  (syntax-rules ()
+;;    ((_ lambda-arg-name body-form list)
+;;     (yail-list-map (lambda (lambda-arg-name) body-form) list))))
+
+(define-syntax filter_nondest
+  (syntax-rules ()
+    ((_ lambda-arg-name body-form list)
+     (yail-list-filter (lambda (lambda-arg-name) body-form) list))))
+
+;;(define-syntax filter_dest
+;;  (syntax-rules ()
+;;    ((_ lambda-arg-name body-form list)
+;;     (yail-list-filter! (lambda (lambda-arg-name) body-form) list))))
+
+(define-syntax reduceovereach
+  (syntax-rules ()
+    ((_ initialAnswer lambda-arg1-name lambda-arg2-name body-form list)
+      (yail-list-reduce initialAnswer (lambda (lambda-arg1-name lambda-arg2-name) body-form) list))))
+
+(define-syntax sortcomparator_nondest
```

Comment:
Were there definitions for `sortcomparator_dest` and `sortkey_dest` in Soojin's branch?  If so, could they, and any other procedures that they use, be put in this file and commented out, similar to `map_dest` and `filter_dest`, above?

Reply:
Fixed